pkg/reconciler/shared/hash/hash.go

```diff
@@ -17,23 +17,36 @@ limitations under the License.
 package hash
 
 import (
+	"crypto/md5"
 	"crypto/sha256"
 	"encoding/json"
 	"fmt"
 
 	"golang.org/x/mod/sumdb/dirhash"
 )
 
-// Compute generates an unique hash/string for the
-// object pass to it.
+// Compute generates an unique hash/string for the object pass to it.
+// with sha256
 func Compute(obj interface{}) (string, error) {
-	h := sha256.New()
 	d, err := json.Marshal(obj)
 	if err != nil {
 		return "", err
 	}
-	h.Write(d)
-	return fmt.Sprintf("%x", h.Sum(nil)), nil
+	hashSha256 := sha256.New()
+	hashSha256.Write(d)
+	return fmt.Sprintf("%x", hashSha256.Sum(nil)), nil
+}
+
+// Compute generates an unique hash/string for the object pass to it.
+// with md5
+func ComputeMd5(obj interface{}) (string, error) {
```

Comment:
I don't see usage of this function
Any reason to keep it ?

Reply:
https://github.com/tektoncd/operator/pull/1656/files#diff-b406f63a0594166033d9b8126a8a033c2fbe2eb7483c732a97ac1fd97fe1ef3bR391